### PR TITLE
test/unittest_bluefs: When fsync ret is less than 0, fsync can not be…

### PR DESCRIPTION
**Pull request by ShiqiCooperation**
… continued

Signed-off-by: shiqi <1454927420@qq.com>

### CODE REVIEW

File: src/test/objectstore/test_bluefs.cc [hunk 1]
```diff
@@ -125,7 +126,9 @@ TEST(BlueFS, small_appends) {
     ASSERT_EQ(0, fs.open_for_write("dir", "file_sync", &h, false));
     for (unsigned i = 0; i < 1000; ++i) {
       h->append("abcdeabcdeabcdeabcdeabcdeabc", 23);
-      fs.fsync(h);
+      r=fs.fsync(h);
+      if(r<0)
+       break;
```
Comment (liewegas) [inline src/test/objectstore/test_bluefs.cc]: I think we just want ASSERT_EQ(0, r); here?

**Comment by ShiqiCooperation** [comment]
ok! changes done.

**Comment by varadakari** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Comment by ShiqiCooperation** [comment]
detials shows:HTTP ERROR 404

**Comment by varadakari** [comment]
retest this please

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-19_19:32:29-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**